bin/deploy-release

```diff
@@ -25,6 +25,9 @@ echo "::endgroup::"
 cluster_name=$(terraform -chdir="infra/${app_name}/service" output -raw service_cluster_name)
 service_name=$(terraform -chdir="infra/${app_name}/service" output -raw service_name)
 echo "Wait for service ${service_name} to become stable"
-aws ecs wait services-stable --cluster "${cluster_name}" --services "${service_name}"
+if ! aws ecs wait services-stable --cluster "${cluster_name}" --services "${service_name}"; then
+  echo "First attempt to wait for service stability failed, retrying..."
```

Comment:
Nit: grammar a bit awkward, couple of suggestions to keep it simple:
option A: Repeat "Wait for service app-dev to become stable" (aws ecs wait service-stable already has an error message)
option B: Say "Retrying"